### PR TITLE
RN-928: Tupaia Api to S3 File Upload

### PR DESCRIPTION
**Pull request by chris-pollard**
### Changes:

- Update file upload function.

---

### CODE REVIEW

**Comment by rohan-bes** [comment]
@chris-pollard I think this one will have clashes with the upgrade to the aws sdk version change that I just merged. Wanna do a walk-n-talk review on this and we can resolve the clashes together?

**Comment by chris-pollard** [comment]
updated @rohan-bes

File: packages/utils/src/s3/S3Client.js [hunk 1]
```diff
@@ -71,20 +69,19 @@ export class S3Client {
...
    * @public
-   * @param {string} filePath
+   * @param {string} fileName
+   * @param {*} base64EncodedFile
    * @returns
    */
-  async uploadFile(filePath) {
-    const fileName = path.basename(filePath);
-    const s3FilePath = `${getS3UploadFilePath()}${getUniqueFileName(fileName)}`;
+  async uploadFileFromBase64(fileName, base64EncodedFile) {
```
Comment (rohan-bes) [inline packages/utils/src/s3/S3Client.js:76]: Just a small thing on the naming of this, that I think this works with base64encoded or streams yeah?

Maybe just:
```
  /**
   * @public
   * @param {string} fileName
   * @param {*} fileStream | base64EncodedFile
   * @returns
   */
  async uploadFileFromBase64(fileName, file)
```

**Review by rohan-bes — APPROVED** [review]
LGTM @chris-pollard

Probs worth re-testing to ensure everything works well with the new `lib-storage` method of uploading files. One comment on naming but pre-approving

File: packages/utils/src/s3/S3Client.js [hunk 2]
```diff
@@ -71,20 +69,19 @@ export class S3Client {
...
    * @public
-   * @param {string} filePath
+   * @param {string} fileName
+   * @param {*} file
    * @returns
    */
-  async uploadFile(filePath) {
-    const fileName = path.basename(filePath);
-    const s3FilePath = `${getS3UploadFilePath()}${getUniqueFileName(fileName)}`;
+  async uploadFile(fileName, file) {
```
Comment (chris-pollard) [inline packages/utils/src/s3/S3Client.js:76]: Simplied naming here. @rohan-bes does that work?

**Review by rohan-bes — APPROVED** [review]

**Pull request merged**